src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const calls = (a) => {
+    const sum = (b) => {
+      count++;
+
+      if (count % 2 !== 0 || count <= 3) {
+        return a + b;
+      }
+
+      return 'Bzzz... Error!';
```

Comment:
you'd rather inverse your if-statement
```
if (...) { // if something goes wrong
  // we stop doing what we're doing
}

// otherwise – we do what we need
```
this pattern looks more appropriate